Doc/faq/design.rst

```diff
@@ -401,7 +401,7 @@ Why are there separate tuple and list data types?
 
 Lists and tuples, while similar in many respects, are generally used in
 fundamentally different ways.  Tuples can be thought of as being similar to
-Pascal records or C structs; they're small collections of related data which may
+Pascal ``records`` or C ``structs``; they're small collections of related data which may
 be of different types which are operated on as a group.  For example, a
 Cartesian coordinate is appropriately represented as a tuple of two or three
 numbers.
```

Comment:
I can't reach it, but you could change ``os.listdir('.')`` to `` :func:`os.listdir('.') <os.listdir>` ``.